content/_releases/22/28.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Product Update - v22.28
+description: Details of the product update v22.28 from 14th of July 2022.
+layout: rel
+version: "22.28"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-07-14
+redirect_from:
+  - /releases/22.28.html
+---
+
+## Improvements and Updates
+
+### Per-tenant frontend styles feature removed
+
+We've removed the feature that builds per-tenant styles and applied them based on `tenant.cssEnabled` and `tenant.code property`. We also removed all tenant styles from our codebase. Please use `tenant.customStylesheets` instead.
+
+> PLease visit our [API documentation]({{site.data.tenant.apiBaseUri}}/docs/v2/#tenants) for more information.
+
+###  Mailchimp is no longer supported
```

Comment:
Needs explanation - see prev comment

Reply:
@KMobbs what explanation do you expect here? All existing clients already use customStylesheets for styles (e.g. Squiz). We removed the part of platform which was implemented for old customers (e.g. Telekom). So, we just remove unsuded code.

Reply:
The same for Mailchimp. Our platform oi longer support it, no one uses it. we just clear the platform code

Reply:
OK for me now that this is stated. In general if we make changes that affect noone then I dont understand why they are in release notes. If we do put something in release notes then the relevance of the change to the Users/readers should be immediately clear.
As a user I dont care if elastic.io is doing housekeeping, especially if we remove code that noone uses.  - I only care if I CAN or SHOULD/MUST do something differently.

If we take this attitude:
We've removed the feature that builds per-tenant styles and applied them based on `tenant.cssEnabled` and `tenant.code property`. We also removed all tenant styles from our codebase. Please use `tenant.customStylesheets` instead.

BECOMES:
Tenant UI customisation can now be easily defined using the property `tenant.customStylesheets`.

Reply:
The problem is that 

> Tenant UI customisation can now be easily defined using the property tenant.customStylesheets.

it's not this release feature, so we can't say `now`

Maybe we can just mention smth like `we are working to clean up  (improve?) our code base and removed unsupported parameters from tenants api. please see the api-docs. No actions from your side is required`